translations/tr/CONTRIBUTING.md

```diff
@@ -0,0 +1,155 @@
+```
+
+Sorunu yeniden oluşturun ve sistemin çökmesine izin verin. Çökmeden sonra C:\Windows\MEMORY.DMP dosyasını bulun. Bu dosyayı secure@microsoft.com adresine gönderin. E-posta içinde:
+
+- GitHub sorun numarası (varsa)
+- WSL ekibine gönderildiği bilgisi
+
+olmalıdır.
+
+### 11) WSL Süreç Çökmesini Bildirme
+
+Bir WSL süreci çökmesini bildirmenin en kolay yolu [kullanıcı modu çökme dökümü toplamaktır](https://learn.microsoft.com/windows/win32/wer/collecting-user-mode-dumps).
+
+Çalışan tüm WSL işlemlerinin dökümlerini toplamak için, lütfen yönetici ayrıcalıklarına sahip bir PowerShell istemi açın, günlük dosyalarınızı koymak istediğiniz bir klasöre gidin ve aşağıdaki komutları çalıştırın: 
+
+```
+Invoke-WebRequest -UseBasicParsing "https://raw.githubusercontent.com/microsoft/WSL/master/diagnostics/collect-wsl-logs.ps1" -OutFile collect-wsl-logs.ps1
+Set-ExecutionPolicy Bypass -Scope Process -Force
+.\collect-wsl-logs.ps1 -Dump
+```
+
+Kod tamamlandığında günlük dosyasına giden yolun çıktısını verecektir.
+
+#### Otomatik Çökme Dökümü Toplamayı Etkinleştirme
+
+Çökmeniz düzensizse veya yeniden oluşturması zorsa, lütfen bu davranışa ilişkin günlükleri yakalamak için otomatik çökme dökümlerini etkinleştirin:
+
+```
+md C:\crashes
+reg.exe add "HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\Windows Error Reporting\LocalDumps" /f
+reg.exe add "HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\Windows Error Reporting\LocalDumps" /v DumpFolder /t REG_EXPAND_SZ /d C:\crashes /f
+reg.exe add "HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\Windows Error Reporting\LocalDumps" /v DumpType /t REG_DWORD /d 2 /f
+```
+
+Daha sonra çökme dökümleri otomatik olarak C:\crashes dosyasına yazılacaktır.
+
+İşiniz bittiğinde, aşağıdaki komutu yükseltilmiş bir komut isteminde çalıştırarak çökme dökümü toplamayı devre dışı bırakabilirsiniz:
+
+```
+reg.exe delete "HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\Windows Error Reporting\LocalDumps" /f
+```
+
+### 12) wslservice İçin Time Travel Debugging İzleri Toplama
+
+Time Travel Debugging İzleri'ni toplamak için:
+
+1) [Windbg Preview'i yükleyin](https://apps.microsoft.com/store/detail/windbg-preview/9PGJGD53TN86?hl=en-us&gl=us&rtc=1)
+
+2) Yönetici olarak açtığınız komut isteminde `windbgx` komutunu çalıştırarak windbg önizlemesini yönetici olarak açın
+
+3) `file` -> `Attach to process` seçeneğine gidin.
+
+4) `Record with Time Travel Debugging` seçeneğini işaretleyin (sağ altta)
+
+4) `Show processes from all users` seçeneğini işaretleyin (en altta)
+
+5) `wslservice.exe` dosyasını seçin. Not, eğer wslservice.exe çalışmıyorsa, onu şu şekilde başlatabilirsiniz: `wsl.exe -l`
+
+6) `Configure and Record`'a tıklayın (izler için seçtiğiniz klasörü not edin)
+
+7) Sorunu yeniden oluşturun
+
+8) windbg'ye geri dönün ve `Stop and Debug`'a tıklayın
+
+9) İz toplama işlemi tamamlandığında, `Stop Debugging` seçeneğine tıklayın ve Windbg'yi kapatın
```

Comment:
```suggestion
9) İz toplama işlemi tamamlandığında, `Stop Debugging` seçeneğine tıklayın ve WinDbg'yi kapatın
```

---

I can't speak to Turkish style, you might want to put `` ` `` around `WinDbg`...